gtd-timer/Controllers/LogInController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+using gtdtimer.Services;
+using gtdtimer.Timer.DTO;
+using gtdtimer.Timer.DAL.UnitOfWork;
+
+namespace gtdtimer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LogInController : ControllerBase
+    {
+        private ILogInService logInService;
+        private IUnitOfWork unitOfWork;
```

Comment:
redundant

Reply:
Deleted variable of type IUnitIfWork from LogInController